lastMeet.php

```diff
@@ -0,0 +1,16 @@
+<?php session_start(); ?>
+<html>
+
+<head>
+  <title>13th Annual Alumni Meet &middot; Home</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="icon" href="img/logo-square.png" media="screen">
+  <link rel="stylesheet" href="css/bootstrap.min.css" media="screen">
+  <link rel="stylesheet" href="css/main.css" media="screen">
+</head>
+
+<body>
+  <?php include "navbar.php"; ?>
+  <img src="img/meet.jpg" alt="" width="100%" />
```

Comment:
put this inside a `div.container-fluid` element. And remove the inline CSS of width 100%. Bootstrap will take care of that.